Functions/FSL/xASL_fsl_SetFSLdir.m

```diff
@@ -55,7 +53,14 @@
     FSLdir = x.FSLdir;
     RootWSLdir = x.RootFSLdir;
     return;
-elseif isfield(x, 'FSLdir') && ~isfield(x, 'RootFSLdir') && ~ispc
+end
+
+% For VABY quantification, we cannot run automatic detection
+if isfield(x, 'external') && isfield(x.external, 'ExternalQuantificationType') && strcmp(x.external.ExternalQuantificationType, 'VABY')
```

Comment:
This is very confusing to the user, using the FSL path for VABY.
What about cases where we use FSL TopUp + VABY quantification?

Reply:
Yes of course. This is related to all the comments above. Once we decide to split VABY as a separate thing and not as a variant of FSL, we can treat it as a fully separate function.